src/main/java/com/iridium/iridiumskyblock/configs/Permissions.java

```diff
@@ -14,7 +14,7 @@
     public Permission blockPlace = new Permission("blockPlace", new Item(XMaterial.GRASS_BLOCK, 11, 1, "&bPlace Blocks", Arrays.asList("&7Grant the ability to place any blocks on your island.", "", "&b&lPermission", "{permission}")), IslandRank.MEMBER);
     public Permission bucket = new Permission("bucket", new Item(XMaterial.BUCKET, 12, 1, "&bUse Buckets", Arrays.asList("&7Grant the ability to fill and empty buckets on your island.", "", "&b&lPermission", "{permission}")), IslandRank.MEMBER);
     public Permission doors = new Permission("doors", new Item(XMaterial.OAK_DOOR, 13, 1, "&bUse Doors", Arrays.asList("&7Grant the ability to use doors or trapdoors on your island.", "", "&b&lPermission", "{permission}")), IslandRank.MEMBER);
-    public Permission redstone = new Permission("redstone", new Item(XMaterial.REDSTONE, 14, 1, "&bUse Redstone", Arrays.asList("&7Grant the ability to use buttons, levels or pressure plates on your island.", "", "&b&lPermission", "{permission}")), IslandRank.MEMBER);
+    public Permission redstone = new Permission("redstone", new Item(XMaterial.REDSTONE, 14, 1, "&bUse Redstone", Arrays.asList("&7Grant the ability to use buttons, levels, or pressure plates on your island.", "", "&b&lPermission", "{permission}")), IslandRank.MEMBER);
```

Comment:
There a typo, should be "levers" not "levels" @PeachesMLG  @RyVador

Reply:
Peaches pls fix D: